### PR TITLE
homepage created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**
preview :- https://vide-x.netlify.app/

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Small PR 
* Looks Good

File: src/components/BannerVideo/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+const BannerVideo = ({ data }) => {
+  const [play, setPlay] = useState(false);
+  const videoRef = useRef();
+
+  const togglePlay = () => setPlay((prev) => !prev);
+
+  useEffect(() => {
+    if (play) videoRef.current.play();
+    else videoRef.current.pause();
```
Comment (rahulyadav139) [inline src/components/BannerVideo/index.js:12]: Suggestion: you can add videoRef as in the dependency array.

**Review by rahulyadav139 — APPROVED** [review]
Great work. though it is a small PR, clean code.

File: src/components/BannerVideo/index.js (hunk 1 above)
Comment (ivinayakg) [reply src/components/BannerVideo/index.js:12]: wont need it, as it wont make a difference

**Pull request merged**